Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/ConvertToWavelength.py

```diff
@@ -0,0 +1,146 @@
+# pylint: disable=too-few-public-methods
+
+""" ConvertToWavelength algorithm converts to wavelength units."""
+
+from __future__ import (absolute_import, division, print_function)
+from mantid.kernel import (Direction, StringListValidator, Property)
+from mantid.dataobjects import EventWorkspace
+from mantid.api import (DataProcessorAlgorithm, MatrixWorkspaceProperty, AlgorithmFactory, PropertyMode, Progress)
+from sans.common.constants import EMPTY_NAME
+from sans.common.general_functions import (create_unmanaged_algorithm, append_to_sans_file_tag,
+                                           get_input_workspace_as_copy_if_not_same_as_output_workspace)
+from sans.common.enums import (RebinType, RangeStepType)
+
+
+class ConvertToWavelength(DataProcessorAlgorithm):
+    def category(self):
+        return 'SANS\\Wavelength'
+
+    def summary(self):
+        return 'Convert the units of a SANS workspace to wavelength'
+
+    def PyInit(self):
+        # Workspace which is to be masked
+        self.declareProperty(MatrixWorkspaceProperty("InputWorkspace", '',
+                                                     optional=PropertyMode.Mandatory, direction=Direction.Input),
+                             doc='The workspace which is to be converted to wavelength')
+
+        self.declareProperty('WavelengthLow', defaultValue=Property.EMPTY_DBL, direction=Direction.Input,
+                             doc='The low value of the wavelength binning.')
+        self.declareProperty('WavelengthHigh', defaultValue=Property.EMPTY_DBL, direction=Direction.Input,
+                             doc='The high value of the wavelength binning.')
+        self.declareProperty('WavelengthStep', defaultValue=Property.EMPTY_DBL, direction=Direction.Input,
+                             doc='The step size of the wavelength binning.')
```

Comment:
Would it make sense to have the binning parameters in the standard Mantid format, that is, as a list of double values `[min, step, max]`?